README.md

```diff
@@ -102,7 +102,8 @@ The following packages are optional:
 * **ttf-freefont**
 
  More font support in the reports
-
+ For use genealogic symbols you must have FreeSerif or DejaVu Sans fonts (or other) installed.
```

Comment:
This looks wrong. It gets appended to the previous line which doesn't end with a full stop.

Also DejaVu Sans is not included in this package.

Reply:
I changed this myself.

Reply:
1. I check Gramps on fresh Linux system some years ago.
FreeSerif font had better MARRIAGE symbol - big and clear.
DejaVu - had this symbol too, but this symbol - small. 
2. Thank you.